peachjam/templates/peachjam/layouts/main.html

```diff
@@ -29,8 +29,12 @@
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         gtag('js', new Date());
-
-        gtag('config', '{{ PEACHJAM_SETTINGS.google_analytics_id }}');
+      // split multiple ids into array
+      const google_analytics_ids = '{{ PEACHJAM_SETTINGS.google_analytics_id }}'.split(',').map(id => id.trim());
```

Comment:
If the tokens don't have spaces then Splitting on spaces (rather than commas) is always simpler and less error prone. Please also update the settings object help text to indicate that multiple values are supported.

Reply:
Done :+1: